scripts/prevalence_helper.sh

```diff
@@ -10,23 +15,45 @@ if ! command -v gh &> /dev/null; then
   exit
 fi
 
+# Set defaults
+COMMIT_AND_PULL_REQUEST=0
+STAY_ON_CURRENT_BRANCH=0
+
+while getopts "bc" OPTION; do
+    case $OPTION in
+    b)
+        STAY_ON_CURRENT_BRANCH=1
+        ;;
+    c)
+        COMMIT_AND_PULL_REQUEST=1
+        ;;
+    *)
+        ;;
+    esac
+done
+
+# Load the API key
 KEYSTORE=".secure-keys"
 source "$KEYSTORE"
 if [ ! -f "$KEYSTORE" ]; then
   echo "$KEYSTORE does not exist. Please obtain an API key from https://apidocs.covidactnow.org/"
   exit
 fi
 
-# Update local branch
-git checkout main
-git pull
+if [[ $STAY_ON_CURRENT_BRANCH != 1 ]]; then
+  echo "Creating auto-update-prevalence branch based on main"
+  # Update local branch
+  git checkout main
+  git pull
+
+  # Delete the remote branch. It usually doesn't exist unless the script failed last time.
+  git push origin --delete auto-update-prevalence &> /dev/null || true
```

Comment:
Curious, what was your reasoning for this to always use the same branch as opposed to, say, "auto-update-prevalence-$(date)"?

Reply:
Oh just hadn't thought of it. That works!

Reply:
I still think it might be good to delete the branch, though. In case we're run into problems and need to run the script multiple times. Either that or add hour/minute to the branch name. Thoughts?

Reply:
i went with hour/minute/secon